### PR TITLE
Add AWS Textract to the list of AWS services Datadog provides traces for

### PR DESCRIPTION
**Pull request by Sticksword**
## Goal & Description
My team at Square uses Datadog for tracing. We currently use the Datadog Ruby SDK to provide metrics for our S3 and DynamoDB integrations. We would like to also use Datadog to provide metrics for our upcoming [Textract](https://aws.amazon.com/textract/) integration. After taking a look at their [Textract SDK](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/Textract.html), it seems like they also use the AWS Seahorse Client to make calls to their endpoints. I figured we can add `Textract` to the list of services traced by Datadog and things should "just work". We were hoping to have metrics like latency and whatnot for Textract in our Datadog AWS dashboard. Currently we see metrics for S3 and DynamoDB.

## Test Coverage
It seems after I made my change, all main specs are passing:
```
bundle exec rake spec:main
...
=> 2014 examples, 0 failures, 31 pending
```

### CODE REVIEW

**Review by ericmustin — APPROVED** [review]
👋 Hey @Sticksword , thanks for the contribution. I think the changes here LGTM, i agree things should _just work_. Given the way AWS adds new services at a pretty fast clip. I think we might want to schedule some future work around making the `SERVICE` List safely extendable via config option so that y'all aren't stuck waiting on a new version every time you want to use a new AWS gizmo.

**Comment by Sticksword** [comment]
Sounds good, thanks @ericmustin!

**Comment by Sticksword** [comment]
Hmm not sure where the merge button is, I thought after an approval I could merge.

**Comment by Sticksword** [comment]
Could you merge this PR? I don't think I have permission now that I think of it.

**Comment by marcotc** [comment]
Thank you again for your work in this PR, @Sticksword. We've just released [v0.44.0](https://github.com/DataDog/dd-trace-rb/releases/tag/v0.44.0), which includes these changes. Let us know if you have any feedback with this new version.

**Pull request merged**
